x/mock/simulation/random_simulate_blocks.go

```diff
@@ -155,7 +155,7 @@ func createBlockSimulator(testingMode bool, tb testing.TB, t *testing.T, event f
 					AssertAllInvariants(t, app, invariants, log)
 				}
 				if opCount%50 == 0 {
-					fmt.Printf("\rSimulating... block %d/%d, operation %d/%d.", header.Height, totalNumBlocks, opCount, blocksize)
+					fmt.Printf("\rSimulating... block %d/%d, operation %d/%d.  ", header.Height, totalNumBlocks, opCount, blocksize)
```

Comment:
Spaces added since you can get in weird positions otherwise where you saw the `.` and the previous prints numbers.